ocaml/sdk-gen/go/gen_go_helper.mli

```diff
@@ -30,3 +30,74 @@ module Json : sig
 
   val api_errors : Mustache.Json.value list
 end
+
+module Types : sig
```

Comment:
The module name may better be `TypesOfMessages`

Reply:
Renamed to `TypesOfMessages`.